build.gradle

```diff
@@ -54,7 +54,7 @@ node {
 }
 
 ext {
-    perftop_version = '1.0.0'
+    perftop_version = '1.3.0'
```

Comment:
Does this need an update if there isnt anything new? I'm seeing 1.0.0 .. so wondering if this versioning needs to b in sync with OpenSearch.
@kjoseph07 , @sruti1312  ?

Reply:
```perftop_version``` is not used anywhere. I think we can remove this variable. 

version for perftop is parsed directly from opensearch_version. 
```version = opensearch_version - '-SNAPSHOT' + '.0'```